.github/workflows/production-stage.yml

```diff
@@ -63,14 +64,27 @@ jobs:
           echo "parsed tag: $TAG"
           echo "image-tag=$TAG" >> $GITHUB_OUTPUT
 
+      - name: Setup Node.js
+        uses: actions/setup-node@v4.0.2
+        with:
+          node-version: '20.11.1'
+      
+      - name: Install Minimal Dependencies
+        working-directory: ./.github
+        run: npm install
+        
+      - name: Switch to Production Env
+        working-directory: ./.github
+        run: npm run import-meta-env:production
+      
```

Comment:
if you rebuild the container when you push it to production, you can't ensure that the build from the development phase will work in production. => "npm install" always installs the latest packages!

Reply:
Here, `npm install` is used in a different context. I added a new [package.json](https://github.com/OlympGuide/Frontend/blob/c7def169e2a422517a8f5b7c249c3e5785f91ce9/.github/package.json) file in the /.github folder containing a minimal setup to replace the env variables. This does not rebuild the container but modifies solely one line in the index.html file that defines the env variables for the website.